lib/bundles/codemirror.js

```diff
@@ -34,12 +34,29 @@
 // Codemirror base lib
 
 import {EditorView, basicSetup} from "codemirror";
-import {css} from "@codemirror/lang-css"
+import {css} from "@codemirror/lang-css";
+import {json} from "@codemirror/lang-json";
+import {sql} from "@codemirror/lang-sql";
+import {highlightTree} from "@lezer/highlight";
+import {defaultHighlightStyle} from "@codemirror/language";
 
 window.CodeMirror = {
     EditorView: EditorView,
     setup:      basicSetup,
     languages:  {
         css: css,
+        json: json,
+        sql: sql,
     },
+    defaultHighlightStyle: defaultHighlightStyle,
+    runMode: (textContent, language, callback) => {
```

Comment:
Seems to be inspired by https://discuss.codemirror.net/t/static-highlighting-using-cm-v6/3420

```suggestion
    runMode: (textContent, language, callback) => {
        // see https://discuss.codemirror.net/t/static-highlighting-using-cm-v6/3420
```